src/main/java/equipments/technology/ArmyComputer.java

```diff
@@ -0,0 +1,10 @@
+package equipments.technology;
+
+public class ArmyComputer extends AbstractTechnologyEquipment {
+    public ArmyComputer(String IP) {
+        this.IP = IP;
```

Comment:
is this logic unique only to an `ArmyComputer`? if the answer is no- move it to a shared parent class